src/woo_publications/publications/models.py

```diff
@@ -48,6 +48,12 @@ def __str__(self):
 
 
 class Document(models.Model):
+    uuid = models.UUIDField(
+        _("UUID"),
+        unique=True,
+        default=uuid.uuid4,
+        editable=False,
+    )
```

Comment:
There will be a Sentry error coming your way soon when I deploy this :-) I'll let it happen as a learning experience.

Reply:
I am very intrigued and scared at the same time.